index.html

```diff
@@ -2,9 +2,20 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>AmericanScraps</title>
 </head>
 <body>
-
+    <div class="wrapper">
+        <div class="header">
+            <div class="inner bound">
+                <ul class="menu">
+                    <li><a href="">About</a></li>
+                    <li><a href="">Shop</a></li>
+                </ul>
+                <div class="logo"></div>
+                <div class="icons"></div>
```

Comment:
what is class?

Reply:
it is the class of social network such as twitter, facebook, instagram and etc.

Reply:
ok, but where is it described? css ? js ?